packages/sdk/src/Fuel.test.ts

```diff
@@ -132,3 +133,42 @@ describe('Fuel', () => {
     await response.wait();
   });
 });
+
+describe('Fuel Events', () => {
+  beforeAll(() => {
+    MockConnection.start();
+  });
+
+  test('Events: Connection events', async () => {
+    const handleConnectionEvent = jest.fn();
+    window.addEventListener(FuelWalletEvents.CONNECTION, handleConnectionEvent);
```

Comment:
```suggestion
    fuel.on(FuelWalletEvents.CONNECTION, handleConnectionEvent);
```

Reply:
Looking at the mock implementation, `fuel.on` should work correctly

Reply:
I thought so too, but it didn't work locally.